.github/workflows/bump-datadog-ci.yml

```
@@ -74,6 +74,8 @@ jobs:
           script: |
             const body = `This PR was automatically created because a new version of datadog-ci was published.
 
+            In case the ci/bitrise check is failing, please go to the `Details` and then click `Rebuild` and rebuild the unsuccessful workflows.
```

You can't use backticks here because you are already in a multiline backtick string.
```suggestion
            In case the Bitrise CI check is failing, please follow the "Details" link in GitHub and then, in Bitrise, click "Rebuild" and choose "Rebuild unsuccessful Workflows".
```